.github/workflows/release-page.yml

```diff
@@ -136,3 +139,28 @@ jobs:
             major=$(echo ${version} | cut -d'.' -f1)
             minor=$(echo ${version} | cut -d'.' -f2)
             glrd --type stable,patch,dev --version ${major}
+
+  # Run glvd ingestion *after* glrd release was triggered, but *before* release page is generated
+  # This is needed for the automatic changelog generation
+  glvd:
+    if: ${{ inputs.type }} == 'patch'
+    name: Update GLVD distro list
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      # This is required for requesting the JWT
+      id-token: write
+    needs:
+      - glrd
+    steps:
+      - name: Get OIDC token
+        id: get-token
+        run: |
+          IDTOKEN=$(curl -H "Authorization: bearer $ACTIONS_ID_TOKEN_REQUEST_TOKEN" "$ACTIONS_ID_TOKEN_REQUEST_URL&audience=glvd" | jq -r '.value')
+          echo "idToken=${IDTOKEN}" >> $GITHUB_OUTPUT
+      - uses: azure/k8s-set-context@v4
+        with:
+           method: kubeconfig
+           kubeconfig: "${{ secrets.KUBECONFIG }}"
```

Comment:
This secret is not yet created, and the repo also needs to have permission for the OIDC token in the cluster